src/sentry/tasks/integrations/github/pr_comment.py

```diff
@@ -309,6 +309,12 @@ def github_comment_reactions():
                 metrics.incr("github_pr_comment.comment_reactions.rate_limited_error")
                 break
 
+            if e.code == 404:
```

Comment:
What other error codes can be thrown from here?

Reply:
the one we want to check for is if the comment has been deleted (presumably) which is why the API is returning 404 not found. this is happening quite frequently so we are checking for this one specifically and not capturing the exception, but for other errors we will still capture the exception